x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -71,22 +58,7 @@ export function ServiceOverview() {
       end={end}
     >
       <ChartPointerEventContextProvider>
-        {isPendingServiceAgent ? (
-          <EuiFlexGroup justifyContent="center">
-            <EuiFlexItem grow={false}>
-              <EuiSpacer size="l" />
-              <EuiLoadingLogo logo="logoObservability" size="l" />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        ) : (
-          <>
-            {isMobileAgent ? (
-              <ServiceOverviewMobileCharts {...serviceOverviewProps} />
-            ) : (
-              <ServiceOverviewCharts {...serviceOverviewProps} />
-            )}
-          </>
-        )}
+        <ServiceOverviewCharts {...serviceOverviewProps} />
```

Comment:
Same as above, maybe we could render all charts and tables on this component directly.